common/models/wordbreakers/test/test-default-word-breaker.js

```diff
@@ -32,4 +32,155 @@ describe('The default word breaker', function () {
+    let words2 = breaks2.map(span => span.text);
+
+    assert.deepEqual(words2, [`.`, `drop`, `:`, `:`, `the`, `'`, `'`, `extras`, `.`]);
+
+    // ',': MidNum (is NOT included by rule!)
+    let breaks3 = breakWords('do br,eak that');
+    let words3 = breaks3.map(span => span.text);
+
+    assert.deepEqual(words3, ['do', 'br', ',', 'eak', 'that']);
+  });
+
+  it('treats Hebrew properly (WB7a-c)', function() {
+    const aleph = 'א';
+    const bet = 'ב';
+
+    // As Hebrew is RTL... this is probably the clearest way for us LTR people to
+    // clearly see what's going on without ordering mechanics messing up the render.
+    let breaks = breakWords(`${aleph}' ${aleph}" ${aleph}"${bet}`);
+    let words = breaks.map(span => span.text);
+
+    // A lingering double-quote isn't cool, but one in the middle's fine.
+    // Lingering single-quote is fine regardless.
+    assert.deepEqual(words, [`${aleph}'`, `${aleph}`, `"`, `${aleph}"${bet}`]);
+  });
+
+  it(`doesn't break within digit + digit/letter sequences (WB8-10)`, function() {
+    let breaks = breakWords('a1b2c3 hunter2 ab12cd34 1234567890');
+    let words = breaks.map(span => span.text);
+
+    assert.deepEqual(words, ['a1b2c3', 'hunter2', 'ab12cd34', '1234567890']);
+  });
+
+  it('does not break within formatted number sequences (WB11-12)', function() {
+    // Note:  `'` fits "MidNumLetQ", part of the two rules!
+    let breaks = breakWords(`1.2.3 3,458.01 3.45'8,01`);
+    let words = breaks.map(span => span.text);
+
+    assert.deepEqual(words, [`1.2.3`, `3,458.01`, `3.45'8,01`]);
+
+    let breaks2 = breakWords(`.1' ,3.`);
+    let words2 = breaks2.map(span => span.text);
+
+    assert.deepEqual(words2, [`.`, `1`, `'`, `,`, `3`, `.`]);
+  });
+
+  it('does not break between Katakana (WB13)', function() {
+    const kataSmA = '\u30a2'; //ァ
+    const kataA = '\u30a2'; //ア
+    const kataSound = '\u309b'; // ゛
+
+    let breaks = breakWords(`${kataSound}${kataA} ${kataSmA}${kataSound}b ${kataA}${kataSound}${kataSmA}`);
+    let words = breaks.map(span => span.text);
+
+    assert.deepEqual(words, [
+      `${kataSound}${kataA}`,
+      `${kataSmA}${kataSound}`,
+      'b',
+      `${kataA}${kataSound}${kataSmA}`
+    ]);
+  });
+
+  it('does not break form extenders (WB13a-b)', function() {
+    // The `_` (underscore) fits the ExtendNumLet class this rule focuses on.
+    const kataA = '\u30a2'; //ア
+
+    let breaks = breakWords(`${kataA}_a__0_b_${kataA} _${kataA} 1_ _c_ ____`);
+    let words = breaks.map(span => span.text);
+
+    assert.deepEqual(words, [
+      `${kataA}_a__0_b_${kataA}`,
+      `_${kataA}`,
+      `1_`,
+      `_c_`,
+      `____`
+    ]);
+  });
+
+  it('handles emoji flag sequences properly (WB15-16)', function() {
+    let breaks = breakWords(`🇨🇦🇰🇭🇽🇽`); // May not render well within VSCode.
+    let words = breaks.map(span => span.text);
+
+    assert.deepEqual(words, ['🇨🇦', '🇰🇭', '🇽🇽']); // the last is two emoji in a row.
+  });
```

Comment:
![image](https://user-images.githubusercontent.com/25213402/188581048-c9c661a1-8bf9-4d75-b584-133f5c331db0.png)

Hence why the comments are relevant.

Reply:
Can we replace `🇽🇽` with `\u{1f1fd}\u{1f1fd}` and avoid the confusion?